cmd/rook/main.go

```diff
@@ -232,6 +240,14 @@ func setLogLevel() {
 	capnslog.SetGlobalLogLevel(cfg.logLevel)
 }
 
+func logStartupInfo(cmdFlags *pflag.FlagSet) {
+	// log the version number, arguments, and all final flag values (environment variable overrides
+	// have already been taken into account)
+	flagValues := flags.GetFlagsAndValues(cmdFlags)
+	logger.Infof("starting rook %s with arguments '%s'", version.Version, strings.Join(os.Args, " "))
+	logger.Infof("flag values: %s", strings.Join(flagValues, ", "))
```

Comment:
note this will also log flags that are "secret", such as `--admin-secret`.  I'm wondering if we should filter those, but perhaps that's not a big deal since the logs are restricted to root access on the nodes.